### PR TITLE
SRCH-826 Resolve spec/models/elastic_news_item_spec.rb failures

### PR DESCRIPTION
**Pull request by MothOnMars**

### CODE REVIEW

File: spec/controllers/searches_controller_spec.rb [hunk 1]
```diff
@@ -567,9 +567,7 @@
       expect(assigns[:search]).to be_an_instance_of(NewsSearch)
     end
 
-    # Temporarily disabling these specs during ES56 upgrade
-    # https://cm-jira.usa.gov/browse/SRCH-814
-    xit "should find news items that match the query for the affiliate" do
+    it "should find news items that match the query for the affiliate" do
       get :news, :query => "element", :affiliate => affiliate.name, :channel => rss_feeds(:white_house_blog).id, :tbs => "w"
```
Comment (peggles2) [inline spec/controllers/searches_controller_spec.rb]: Let's use single quotes here.

File: spec/controllers/searches_controller_spec.rb [hunk 2]
```diff
@@ -587,9 +585,7 @@
       it { is_expected.to redirect_to 'https://www.usa.gov/search-error' }
     end
 
-    # Temporarily disabling these specs during ES56 upgrade
-    # https://cm-jira.usa.gov/browse/SRCH-814
-    pending "when the query is blank and total is > 0" do
+    context "when the query is blank and total is > 0" do
       before { get :news, :query => "", :affiliate => affiliate.name, :channel => rss_feeds(:white_house_blog).id, :tbs => "w" }
```
Comment (peggles2) [inline spec/controllers/searches_controller_spec.rb]: Let's use single quotes here.

File: spec/models/news_search_spec.rb [hunk 3]
```diff
@@ -90,9 +90,7 @@ def filterable_search_options
         expect(@search.run).to be true
       end
 
-      # Temporarily disabling these specs during ES56 upgrade
-      # https://cm-jira.usa.gov/browse/SRCH-816
-      xit "should have more than 0 results" do
+      it "should have more than 0 results" do
         @search.run
```
Comment (peggles2) [inline spec/models/news_search_spec.rb]: Let's use single quotes here.

File: spec/models/web_results_post_processor_spec.rb [hunk 4]
```diff
@@ -81,9 +81,7 @@
         @post_processed_results = post_processor.post_processed_results
       end
 
-      # Temporarily disabling these specs during ES56 upgrade
-      # https://cm-jira.usa.gov/browse/SRCH-818
-      xit "should replace Bing's result title with the NewsItem title" do
+      it "replaces Bing's result title with the NewsItem title" do
         expect(@post_processed_results.first['title']).to eq("Title w/o highlighting")
```
Comment (peggles2) [inline spec/models/web_results_post_processor_spec.rb]: Let's use single quotes here.

Comment (MothOnMars) [reply spec/models/web_results_post_processor_spec.rb]: These need to be double quotes due to the apostrophe in `Bing's`.

**Pull request merged**